pdcupdater/tests/handler_tests/test_modules.py

```diff
@@ -66,3 +100,24 @@ def test_create_unreleased_variant(self, pdc):
     @mock_pdc
     def test_handle_created_tree(self, pdc):
         self.handler.handle(pdc, self.state_done_msg)
+
+
+    @mock_pdc
+    @mock.patch('pdcupdater.services.koji_rpms_in_tag')
+    def test_handle_ready_state(self, pdc, koji):
+        koji.side_effect = mocked_koji_from_tag
+
+        variant = {}
+        variant["koji_tag"] = "epel7"
+        variant["modulemd"] = self.modulemd_example
+
+        expected_rpms = [
+            {'srpm_nevra': None, 'epoch': 0, 'version': '1.11',
+             'name': 'dvisvgm', 'release': '1.el7',
+             'srpm_commit_hash': '76f9d8c8e87eed0aab91034b01d3d5ff6bd5b4cb',
+             'srpm_commit_branch': 'f26', 'arch': 'src', 'srpm_name': 'dvisvgm'},
+            {'srpm_nevra': 'rubygem-jmespath-1.1.3-1.el7', 'epoch': 0,
+             'version': '1.1.3', 'name': 'rubygem-jmespath-doc',
+             'release': '1.el7', 'arch': 'noarch', 'srpm_name': 'rubygem-jmespath'}]
```

Comment:
Does this one avoid the if branch that looks like:

```
if xmd_rpm['ref'] != mmd_rpm.ref:
    data["srpm_commit_branch"] = mmd_rpm.ref
```

? If so, cool. If not, I suggest adding another test that doesn't cause that if statement to be ```True```.